PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,9 +8,5 @@ Try to fulfill the following points before the Pull Request is merged:
 - [ ] The PR is responsive on smaller screens.
 - [ ] If the requirements.txt have changed, check if the patches still work
 - [ ] If the new code is well commented
-
-For releases only:
-
```

Comment:
Why do you want to delete these points? (Maybe I just can't remember the reason.)

Reply:
Changes have been made.